src/components/policy/policy_external/CMakeLists.txt

```diff
@@ -60,12 +65,54 @@ set(SOURCES
   ${COMPONENTS_DIR}/rpc_base/src/rpc_base/rpc_base.cc
 )
 
+set(GENERATED_MOBILE_POLICY_TYPES_CPP
```

Comment:
I will admit that I don't necessarily have a better solution at the moment which doesn't involve a massive amount of refactoring for the parsing tool.

Reply:
This solution make cpp code, that currently is hard coded to generated. So it reduce amount of places that should be touched in case if new parameter or new function will be added to RPC. 
Also it generated code, that is needed for RGNSD to work ( get specific enum schema for validation)